packages/router/src/links.tsx

```diff
@@ -51,7 +52,12 @@ const useMatch = (pathname: string, options?: UseMatchOptions) => {
     }
   }
 
-  return matchPath(pathname, location.pathname)
+  return matchPath(
```

Comment:
This is the wrong line to comment on, but GH wouldn't let me comment further up...

The doc comments for `useMatch` are all wrong. Could you please also fix those as part of this PR? Thanks! 🙏

Reply:
Hopefully better now, wasn't too sure which part you meant.

Reply:
Sorry, I should have been mor clear.

The examples now look like this
```
 * Match search params key existence
 * const match = useMatch('/about', ['category', 'page'])
```

But that's not actually how you call it, is it? It's `useMatch('/about', { searchParams: ['category', 'page'] })`, right?

Reply:
Good catch, hopefully better now.